### PR TITLE
first take at velruse stub pyramid app for oauth test

### PR DESCRIPTION
**Pull request by thequbit**

### CODE REVIEW

**Comment by thequbit** [comment]
@ryansb I'm a bit lost on what /login/twitter should return ... can you point me in the right direction?  I'll then clean up, and resubmit the PR.

File: server/mediapublic/tests/velruse_stub/__init__.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+if __name__ == '__main__':
+
+    config = Configurator()
+
+    # configure the routes
+    config.add_route('/auth_info', '/auth_info')
+    config.add_view(auth_info, route_name='/auth_info')
+
+    config.add_route('/login/twitter', '/login/twitter')
```
Comment (ryansb) [inline server/mediapublic/tests/velruse_stub/__init__.py:26]: This endpoint should be redirecting as a POST to /login on the calling application with token as a URL parameter, used [here](https://github.com/mediapublic/mediapublic/pull/51/files#diff-0da2e782bbd17fbf39ed578b6d6a4f50R144). 

The real flow is:
1. User goes to /login, hits button to log in with twitter
2. User is sent to `POST /auth/login/twitter` by the form
3. User is redirected to twitter.com's oauth provider, where they log in (or their existing login is used)
4. Twitter redirects them to `POST <ourapp>/login` with the token parameter
5. We grab the auth_info from velruse using the token

The fake flow should be:
1. User goes to /login, hits form button
2. Stub `POST /auth/login/twitter` redirects back to `POST /login` with fake token parameter
3. mediapublic then uses the token to ask /auth_info for info about the login associated with the token
4. The fake login data is sent back

File: server/mediapublic/tests/velruse_stub/auth_info.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            }
+        ],
+        "photos": [
+            {
+                "value": "http://example.com/images/image.jpg"
+            }
+        ]
+    },
+    "credentials": {
+        "oauthAccessToken": "241840562-dgDsuqWlFzNYPbH2tMmPyH9fg2Zs823c83sfBbD8",
```
Comment (ryansb) [inline server/mediapublic/tests/velruse_stub/auth_info.json:32]: Please tell me these aren't real access tokens.

Comment (thequbit) [inline server/mediapublic/tests/velruse_stub/auth_info.json:32]: I mean, they may be _someones_ real access tokens, but they are random.

File: server/mediapublic/tests/velruse_stub/__init__.py (hunk 1 above)
Comment (thequbit) [inline server/mediapublic/tests/velruse_stub/__init__.py:26]: Awesome.  I'll get that implemented over lunch.

**Pull request closed without merging**